changelog.txt

```diff
@@ -0,0 +1,2 @@
+* Fixed bug in Firestore emulator where some FieldTransforms were sending back incorrect results.
+* Added support for read-only transactions to Firestore emulator.
```

Comment:
This release also seemed to have fixed collection group queries on listen streams.

Reply:
[@firebase/testing does not handle FieldValue.increment #1799](https://github.com/firebase/firebase-js-sdk/issues/1799)  is still not fixed. I'm starting a brand new firebase project today, I still have the extact same proplem with most recent packages installed.

Reply:
Please open a new issue with specific details of what you're trying to do and what's failing.

This specific issue definitely was fixed because it was blocking being able to run the Firestore SDK integration test suite against the emulator. Since this release we've been able to do that successfully.